cdrom/cdrom/urls.py

```diff
@@ -22,6 +22,7 @@
     path("admin/", admin.site.urls),
     path('aulas/', include('aulas.urls')),
     path('profesores/', include('profesores.urls')),
+    path('', include('cursos.urls')),
```

Comment:
```suggestion
    path('cursos', include('cursos.urls')),
```
Dejar URL en blanco solo para la raiz del sitio. Mejor que cada app tenga su propio path en la URL